### PR TITLE
fixed platformio build

### PR DESCRIPTION
**Pull request by NathanMcGuire30**
Some board config define was changed, so I fixed the platformio side to match

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -61,3 +61,16 @@ lib_deps = Wire
            adafruit/Adafruit BusIO@^1.14.1
            adafruit/Adafruit BNO055@^1.6.1
            adafruit/Adafruit Unified Sensor@^1.1.9
+
+[env:feather_m4_hacky_lora_groundstation]
```
Comment (mcm001) [inline platformio.ini:65]: As we get more boards are we documenting what physical hardware these are somewhere? Maybe we need BOARDS.md with pictures

Comment (mcm001) [reply platformio.ini:65]: Also, add this to CI in main.yaml

**Review by mcm001 — CHANGES_REQUESTED** [review]

File: platformio.ini (hunk 1 above)
Comment (NathanMcGuire30) [reply platformio.ini:65]: alright I added it

**Review by NathanMcGuire30 — COMMENTED** [review]
sick

**Review by SamHaggans — APPROVED** [review]

**Review by mcm001 — APPROVED** [review]

**Pull request merged**